bower.json

```diff
@@ -3,7 +3,7 @@
   "description": "YouTube video playback web component.",
   "homepage": "https://googlewebcomponents.github.io/google-youtube",
   "main": "google-youtube.html",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
Is it appropriate to bump version in this commit? I would think this would be done separately and encompass multiple commits/fixes.